src/libdredd/include/libdredd/mutation_replace_unary_operator.h

```diff
@@ -54,8 +56,14 @@ class MutationReplaceUnaryOperator : public Mutation {
   [[nodiscard]] bool IsValidReplacementOperator(
       clang::UnaryOperatorKind op) const;
 
+  std::string GetExpr(clang::ASTContext& ast_context) const;
```

Comment:
Add a comment saying what this does, similar to what you did in MutationReplaceBinaryOperator.

Reply:
Done.